### PR TITLE
Samsung Internet 13.0 is released

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds Samsung Internet 13.0 to the release information. Cherry-picked from #6589.

### CODE REVIEW

**Review by foolip — APPROVED** [review]

File: browsers/samsunginternet_android.json [hunk 1]
```diff
@@ -167,15 +167,21 @@
...
           "engine_version": "79"
         },
         "12.1": {
           "release_date": "2020-07-07",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "79"
+        },
+        "13": {
```
Comment (foolip) [inline browsers/samsunginternet_android.json:180]: Wait, https://github.com/mdn/browser-compat-data/pull/6589 says "13.0", so this wasn't cherry-picked verbatim as the commit message would suggest.

File: browsers/samsunginternet_android.json [hunk 2]
```diff
@@ -167,15 +167,21 @@
...
+          "release_date": "2020-08-28",
+          "status": "retired",
```
Comment (Elchi3) [inline browsers/samsunginternet_android.json:182]:
```suggestion
          "status": "current",
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]

**Review by Elchi3 — APPROVED** [review]
:)

File: browsers/samsunginternet_android.json (hunk 1 above)
Comment (vinyldarkscratch) [reply browsers/samsunginternet_android.json]: (I probably confused this with Safari versions when I was writing it... /)_-)

**Review by foolip — APPROVED** [review]

**Pull request merged**